src/autowire.js

```diff
@@ -19,6 +19,50 @@ function registerRoutes (Vue, requireContext) {
   });
 }
 
+/**
+ * Load router files
```

Comment:
Looks like copy/pasted comments from the router function (same on registerDirectives)